com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 This file documents all notable changes to this package. Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+## [0.1.0] - 2021-06-01
+
+This is hotfix v0.1.1 for the initial experimental Unity MLAPI Package.
+
+### Changes
+
+* Updated the package version to correctly document the published version for MLAPI.
```

Comment:
Maybe something more direct would be helpful to head off questions from the community about this release. Something like `Fixed issue with Unity Registry package version missing some fixes from the 0.1.0 release`?

Reply:
Works for me! Wasn't sure how direct you wanted to be. Updating!